### ISSUE TITLE
:split cannnot open files correctly

### ISSUE THREAD

**Issue by unlsycn** [VSCodeVim/Vim#7830]
**Describe the bug**
The :split (also :vsplit, :vsp) command can split the active editor in the right way, but it does not open the specified file. When I attempted to debug it, I found it seems to **be able to get the correct file path** and `vscode.window.showTextDocument(doc);` not to work.

**Expected behavior**
The file would be opened in the splitted editor.

**Screenshots**
![image](https://user-images.githubusercontent.com/31343548/178677780-082740d4-4620-4e65-bc85-3a5236de8ba4.png)
![image](https://user-images.githubusercontent.com/31343548/178677965-b1a2edb5-1297-4246-a6ed-83f09a5e1458.png)

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: 2022/6/29 version
- VSCode version: 1.70.0-insider
- OS: Windows_NT x64 10.0.22000

**Comment by HumphreyDotSln** [comment]
Here is what returned when I execute `vscode.window.showTextDocument(doc)` manually in Debug Console.
```
Promise {
Symbol(async_id_symbol): 6732
Symbol(destroyed): {
destroyed: false
}
Symbol(trigger_async_id_symbol): 6725
[[PromiseResult]]: undefined
[[PromiseState]]: 'pending'
[[Prototype]]: Promise
}
```

**Comment by HSID** [comment]
The same issue happens to me.

**Comment by unlsycn** [comment]
> The same issue happens to me.

#7877 It works for me.

**Comment by LeoLau94** [comment]
Hi @unlsycn , why is your pull request still open? 
It looks like the right way to use the split and vsplit.
I have the same problem now.

**Comment by unlsycn** [comment]
> Hi @unlsycn , why is your pull request still open? It seems like the right way to use the split and vsplit. I have the same problem now, what do you suggest?

Perhaps this issue cannot be reproduced in the maintainer's machine (it reminds me if it relates to the language environment...)
Compiling my fork to .vsix and installing it locally may be a temporary measure as I have done.

**Comment by sworleys** [comment]
just commenting doesn't work on my windows 10 machine either

```
OS Name	Microsoft Windows 10 Pro
Version	10.0.19045 Build 19045
```

```
vscodevim version: v1.24.3
```

**Comment by gyroknight** [comment]
Seeing the same thing on VSCodeVim v1.24.3 on the latest VS Code Insider build (1.76.0). I can see that the correct file gets opened for a split second so it seems like it gets found correctly, however, the tab immediately reverts to the previous file open and shows the "File: <filename> does not exist." error popup in the lower right corner, resulting in the duplicate file open in the new pane like the OP's screenshot

From the Extension Host output, I see the following message every time `:vsp` or `:sp` is used:
```
2023-02-21 15:24:28.666 [warning] TextEditor is closed/disposed
2023-02-21 15:24:28.666 [warning] TextEditor is closed/disposed
```

**Comment by bmw** [comment]
I'm hitting this too.

**Comment by mc956759** [comment]
I encounter this problems, too. Is uilding the branch and installing locally the only one solution?

**Comment by theuema** [comment]
Same thing here. 

macOS
VSCode Version: 1.80.2 Commit: 2ccd690cbff1569e4a83d7c43d45101f817401dc
Vim Version: v1.25.2

**Comment by J-Fields** [comment]
Can someone please confirm whether this is fixed in `1.26.0`?

**Comment by xty** [comment]
> Can someone please confirm whether this is fixed in `1.26.0`?

I tried just a few days ago and the bug was still present. Now with the the release of `1.26.0`, I'm glad to report the issues has indeed been resolved. #8535 seems to have done the trick.

**Comment by xty** [comment]
> Can someone please confirm whether this is fixed in `1.26.0`?

I tried just a few days ago and the bug was still present. Now with the the release of `1.26.0`, I'm glad to report the issue has indeed been resolved. #8535 seems to have done the trick.
